### ISSUE TITLE
Pathlib: Operator '/' not supported for types 'TypeVar['_P']' and 'Literal['foo']'

### ISSUE THREAD

**Issue by tamuhey** [microsoft/pyright#385]
```python
from pathlib import Path

p = Path().parent / "foo"
```

raises

```
Operator '/' not supported for types 'TypeVar['_P']' and 'Literal['foo']'
```

otherwise the below is passed.

```python
p = Path() / "foo"
```

**Comment by tamuhey** [comment]
I found similar issue: https://github.com/microsoft/pyright/issues/191

**Comment by erictraut** [comment]
Thanks for reporting the bug. I'm able to repro it, and I'll work on a fix when I have a moment.

**Comment by erictraut** [comment]
This will be fixed in the next release.

**Comment by tamuhey** [comment]
thanks!!!

**Comment by erictraut** [comment]
This is now fixed in 1.1.8, which I just published.
